### DISCUSSION TITLE
Plug keystone into existing prisma schema

### DISCUSSION THREAD

**Discussion by doei** [keystonejs/keystone#6299] [Questions] [answered]
The auto-generated admin of keystone looks really good, however, my company (along with countless others I'm sure), cannot afford to simply rebuild their API from scratch with a new framework.

It would be amazing to be able to plug keystone into an existing Prisma schema, to take advantage of the admin generation, it would even work by generating a different app alongside an existing API, as long as the keystone app would consume the exact same Prisma schema as the API.

Is this something you guys are thinking about ?

Thanks for reading 🙂

**Comment by raveling** [comment]
Thanks for sharing your idea @doei 🙂 
This is an interesting idea, but not something we'll pursue in the short to medium term.
Happy to keep this open tho, and see if others add their weight to it.

**Comment by dcousens** [comment, marked as answer]
See https://github.com/keystonejs/keystone/discussions/7233
